core/bootstrap/src/test/java/org/eclipse/dataspaceconnector/system/CoreServicesExtensionTest.java

```diff
@@ -71,6 +72,8 @@ void initialize() {
         expect(context.getService(Vault.class)).andReturn(niceMock(Vault.class)).anyTimes();
         expect(context.getService(eq(PrivateKeyResolver.class))).andReturn(niceMock(PrivateKeyResolver.class));
 
+        context.registerService(eq(ReadinessService.class), isA(ReadinessServiceImpl.class));
+        expectLastCall();
```

Comment:
nit: use mockito? (#384)

Reply:
there is #384, which should introduce mockito. not sure if its better to use the "old" tech here, or have two testing frameworks at a time